src/map.jl

```diff
@@ -58,6 +58,56 @@ struct Map{Ax<:Makie.AbstractAxis} <: AbstractMap
     scale::Float64
 end
 
+function setup_figure_and_axis!(figure::Makie.Figure, axis, ext_target, crs)
+    setup_axis!(axis, ext_target, crs)
+    return figure, axis
+end
+
+function setup_figure_and_axis!(figure::Makie.Figure, axis_kws_nt::NamedTuple, ext_target, crs)
+    axis_kws = Dict(pairs(axis_kws_nt))
+    AxisType = pop!(axis_kws, :type, Axis)
+
+    axis = AxisType(figure[1, 1]; axis_kws...)
+
+    setup_axis!(axis, ext_target, crs)
+
+    return figure, axis
+end
+
+_get_parent_layout(gp::Makie.GridPosition) = _get_parent_layout(gp.layout)
```

Comment:
Feel like these helpers could go in Makie.jl eventually?

Reply:
Yeah they should eventually...for now it's mergeable though